scripts/loud.coffee

```diff
@@ -41,7 +44,12 @@ module.exports = (robot) ->
           res.send "Loud deleted."
         else
           res.send "Couldn't find that loud."
-    
+
+      when 'ban'
+        if data in robot.brain.get('louds')
+          if data not in robot.brain.get('louds_banned')
```

Comment:
I think this should be less conditional

```coffee
robot.brain.get('louds_banned').push(data) if not data in robot.brain.get('louds_banned')
deleteLoud(data)
```

In a world where `deleteLoud` is a thing, because I didn't want to copy paste it here.

:cat: :thought: :question: